pkg/assetfetcher/postgres.go

```diff
@@ -30,9 +30,18 @@ func (f *PostgresPhysicalAssetFetcher) FetchPhysicalAsset(ctx context.Context, i
 	var asset domain.PhysicalAsset
 	var ip sql.NullString
 	var deviceID sql.NullInt64
+	var assetResourceOwner sql.NullString
+	var assetBusinessUnit sql.NullString
+	var assetCustomerID sql.NullInt64
 	err := f.DB.Conn().QueryRowContext(ctx, fetchByIPQuery, ipAddress).Scan(
-		&ip, &asset.ResourceOwner, &asset.BusinessUnit, &asset.Network,
-		&asset.Location, &deviceID, &asset.SubnetID, &asset.CustomerID)
+		&ip, &assetResourceOwner, &assetBusinessUnit, &asset.Network,
+		&asset.Location, &deviceID, &asset.SubnetID, &assetCustomerID)
+	if assetCustomerID.Valid {
+		// if we have a customerID from our query, we'll surely have the rest too:
```

Comment:
We are admittedly trusting IPAM quite a lot.  We trust that IPAM will give us all the right information such that the column values we put in our Postgres tables are guaranteed to be filled, so that handling NullString/NullInt/etc is not necessary here for all columns.

I'm inclined to distrust IPAM and make this more resilient, but not in this PR.